apis/study.py

```diff
@@ -65,8 +65,9 @@ def post(self):
             "required": ["title", "image"],
             "additionalProperties": False,
             "properties": {
-                "title": {"type": "string", "minLength": 1},
-                "image": {"type": "string", "minLength": 1},
+                "title": {"type": "string", "minLength": 1, "maxLength": 300},
+                "acronym": {"type": "string", "maxLength": 14},
```

Comment:
I don't think we should enforce a max length here

Reply:
but acronym can accept only 14 charachters in db, if its not set here, then it can give sqlalchemy.exc.DataError